common/azure-common-test/src/main/java/com/azure/common/test/interceptor/PlayBackInterceptor.java

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.common.test.interceptor;
+
+import com.azure.common.test.models.NetworkCallRecord;
+import com.azure.common.test.models.RecordedData;
+import okhttp3.Interceptor;
+import okhttp3.MediaType;
+import okhttp3.Request;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Iterator;
+import java.util.Map;
+
+import static com.azure.common.test.utils.DataTransformer.applyReplacementRule;
+import static com.azure.common.test.utils.DataTransformer.removeHost;
```

Comment:
Package names in Java should never have upper-case letters.

Reply:
Will fix this.